proto/stream_plan.proto

```diff
@@ -109,6 +117,8 @@ message Barrier {
     PauseMutation pause = 7;
     // Resume the dataflow of the whole streaming graph, only used for scaling.
     ResumeMutation resume = 8;
+    // Throttle specific source exec or chain exec.
+    ThrottleMutation throttle = 10;
```

Comment:
```suggestion
    ThrottleMutation throttle = 9;
```

Reply:
https://github.com/singularity-data/risingwave/blob/e5cb7cdd49132db848520c6e7379e869d380281b/proto/stream_plan.proto#L126 
took the 9